config/locales/en.yml

```diff
@@ -657,70 +657,77 @@ en:
   terms:
     body_html: |
       <h2>Privacy Policy</h2>
-
       <h3 id="collect">What information do we collect?</h3>
 
-      <p>We collect information from you when you register on our site and gather data when you participate in the forum by reading, writing, and evaluating the content shared here.</p>
-
-      <p>When registering on our site, you may be asked to enter your name and e-mail address. You may, however, visit our site without registering. Your e-mail address will be verified by an email containing a unique link. If that link is visited, we know that you control the e-mail address.</p>
+      <ul>
+        <li><em>Basic account information</em>: If you register on this server, you may be asked to enter a username, an e-mail address and a password. You may also enter additional profile information such as a display name and biography, and upload a profile picture and header image. The username, display name, biography, profile picture and header image are always listed publicly.</li>
+        <li><em>Posts, following and other public information</em>: The list of people you follow is listed publicly, the same is true for your followers. When you submit a message, the date and time is stored as well as the application you submitted the message from. Messages may contain media attachments, such as pictures and videos. Public and unlisted posts are available publicly. When you feature a post on your profile, that is also publicly available information. Your posts are delivered to your followers, in some cases it means they are delivered to different servers and copies are stored there. When you delete posts, this is likewise delivered to your followers. The action of reblogging or favouriting another post is always public.</li>
+        <li><em>Direct and followers-only posts</em>: All posts are stored and processed on the server. Followers-only posts are delivered to your followers and users who are mentioned in them, and direct posts are delivered only to users mentioned in them. In some cases it means they are delivered to different servers and copies are stored there. We make a good faith effort to limit the access to those posts only to authorized persons, but other servers may fail to do so. Therefore it's important to review servers your followers belong to. You may toggle an option to approve and reject new followers manually in the settings. Please keep in mind that the operators of the server and any receiving server may view such messages, and that recipients may screenshot, copy or otherwise re-share them.</li>
```

Comment:
"to review servers your followers belong to" is slightly odd wording, i'd say "to review **the** servers your followers belong to".

Reply:
you might want to make sure to say that "authorized persons" here includes the moderator of the server.

Reply:
"In some cases it means they are delivered to different servers ..."
Is it uncertain in which cases posts get delivered to other servers?
My understanding may be incorrect but I what I thought was:
 if those users' accounts are hosted on other Mastodon instances, then the posts they are eligible to read will be copied to the server for their instance, and will be subject to the privacy policy of that instance, which may be more permissive than this one.